contracts/base/Dispatcher.sol

```diff
@@ -308,8 +308,12 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks
                             amount := calldataload(add(inputs.offset, 0x60))
                         }
                         Payments.sweepERC1155(token, recipient.map(), id, amount);
+                    } else if (command == Commands.SEAPORT_V2) {
+                        (uint256 value,) = abi.decode(inputs, (uint256, bytes));
```

Comment:
for gas reasons I now decode in assembly for the other commands... its kind of ugly but more efficient. Look at how the other SEAPORT command is decoded to see what I mean

Reply:
Ah yes I see, I'll change